salt/modules/win_snmp.py

```diff
@@ -0,0 +1,215 @@
+# -*- coding: utf-8 -*-
+'''
+Module for managing SNMP service settings on Windows servers.
+The Windows feature 'SNMP-Service' must be installed.
+
+'''
+
+# Import python libs
+from __future__ import absolute_import
+import logging
+
+# Import salt libs
+from salt.exceptions import SaltInvocationError
+import salt.modules.reg as reg
```

Comment:
Instead of doing this, it may be better to just use `__salt__['reg.<my_func>'](<my_arg>)` where needed in the code. That will allow Salt's loader system to manage the import of the module.